content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -83,10 +83,10 @@ Notice that it may take tens of seconds for the authorization policy to be propa
 
 ## Migrate trust domain without trust domain aliases
 
-1. Install Istio with a new trust domain.
+1. Install Istio with a new trust domain and trust domain aliases.
```

Comment:
This is wrong. This section does NOT use a trust domain alias (see line 84).
```suggestion
1. Install Istio with a new trust domain.
```

Reply:
@howardjohn's comment was referring to the example on line 132. That needs to be changed.

Reply:
Sorry, will update.

Reply:
Done